test/server/cards/01_SOR/events/HeroicSacrifice.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Heroic Sacrifice', function() {
+    integration(function(contextRef) {
+        describe('Heroic Sacrifice\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['heroic-sacrifice'],
+                        groundArena: ['isb-agent', { card: 'wampa', exhausted: true }],
+                        spaceArena: ['tieln-fighter']
+                    },
+                    player2: {
+                        groundArena: ['sundari-peacekeeper', 'atst', { card: 'crafty-smuggler', upgrades: ['shield'] }],
+                    }
+                });
+            });
+
+            it('should draw a card and attack with a unit giving +2/+0 for this attack and defeat it after dealing combat damage', function () {
```

Comment:
Could we add one more test where he does damage into a unit just to be safe, since we haven't written many abilities like this one before